core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,6 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
```

Comment:
I guess part of the reason is that master/client have low memory usage and as a result unlikely to have long timeouts due to gc.

@srowen I'm not sure if this is a good change.